src/Npgsql/TypeMapping/BuiltInTypeHandlerResolver.cs

```diff
@@ -164,12 +164,6 @@ class BuiltInTypeHandlerResolver : TypeHandlerResolver
         { "unknown",  new(NpgsqlDbType.Unknown, "unknown") },
     };
 
-    internal static void ResetMappings()
```

Comment:
No objection, but if I understand things right, the legacy timestamp tests (which use this API) are still problematic after this change, so is it worth removing this? If we move them out to a completely separate test run, we can clean everything up there instead.

Reply:
They're still problematic, but since we do not cache DbType anymore, `ResetMappings` is pointless.